src/Ship.cpp

```diff
@@ -944,6 +954,16 @@ void Ship::UpdateAlertState()
 	}
 
 	bool ship_is_near = false, ship_is_firing = false;
+
+	// sanity check: m_lastAlertUpdate should not be in the future.
+	// reset and re-check if it is.
+	if (m_lastAlertUpdate > Pi::game->GetTime() + 2)
```

Comment:
Out of curiosity, where does the +2 come from?

Reply:
arbitrary. could be +1 but. meh.

Reply:
Shouldn't even need to plus anything since it's using `>`